mod-source-record-manager-server/src/main/java/org/folio/dao/JournalRecordDaoImpl.java

```diff
@@ -161,4 +174,33 @@ private String prepareSortingClause(String sortBy, String order) {
     return format(ORDER_BY_PATTERN, sortBy, order);
   }
 
+  private JobLogEntryDtoCollection mapRowSetToJobLogDtoCollection(RowSet<Row> rowSet) {
+    JobLogEntryDtoCollection jobLogEntryDtoCollection = new JobLogEntryDtoCollection()
+      .withTotalRecords(0);
+
+    rowSet.forEach(row -> {
+      JobLogEntryDto jobLogEntryDto = new JobLogEntryDto()
+        .withJobExecutionId(row.getValue("job_execution_id").toString())
```

Comment:
I think it would be better to extract these fields in constants("source_record_order", "title" and others). Because they are repeating in the default sets too. It will help to avoid typo errors.

Reply:
@VRohach, do you propose to move column names to constants that appear only once as well?

Reply:
@RuslanLavrov  If there are some properties that appear once, I think it would be a better move to constants them also, for the same code style. Moreover, they can be used in DAO in the future (for example in the next endpoint). What do you think?

Reply:
I'll move them